wp-cli-templates/block-edit-js.mustache

```diff
@@ -2,26 +2,42 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
-import { Component } from '@wordpress/element';
+import { BlockIcon } from '@wordpress/block-editor';
+import { Notice, Placeholder} from '@wordpress/components';
+import { useState } from '@wordpress/element';
 
 /**
  * Internal dependencies
  */
 import './editor.scss';
+import icon from './icon';
 
-class {{ className }}Edit extends Component {
+export default function {{ className }}Edit( { attributes, className, setAttributes } ) {
 	/**
 	 * Write the block editor UI.
 	 *
 	 * @returns {object} The UI displayed when user edits this block.
 	 */
-    render() {
-        const { attributes, setAttributes } = this.props;
+	const [ notice, setNotice ] = useState();
 
-        return (
-			<p>{ __( 'Block edit goes here', 'jetpack' ) }</p>
-        );
-    }
-}
+	const setErrorNotice = () => __( 'Put error message here.', 'jetpack' );
```

Comment:
Could you add some docs on how this is supposed to be used? I'm unclear on how it works.